Stripe/STPCustomerContext+Private.h

```diff
@@ -0,0 +1,23 @@
+//
+//  STPCustomerContext+Private.h
+//  Stripe
+//
+//  Created by Ben Guo on 6/9/17.
+//  Copyright © 2017 Stripe, Inc. All rights reserved.
+//
+
+#import <Stripe/Stripe.h>
+
+#import "STPBlocks.h"
+
+@class STPAddress;
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface STPCustomerContext (Private)
+
+- (void)updateCustomerWithShipping:(STPAddress *)shipping completion:(nullable STPErrorBlock)completion;
```

Comment:
👍  I'll rename.

I think we should keep `CustomerContext`'s public API surface really small for now – ideally the `BackendAPIAdapter` methods would be private too, but that's not possible right now. I think if we decide to expose customer methods publicly for standalone use without PaymentContext, we should think more about the use case and maybe expose a different API.